packages/_example/src/frameworks/nest-express-v8.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { NestFactory } from '@nestjs/core';
+
+@Module({ imports: [], controllers: [], providers: [] })
```

Comment:
I think we can add simple route like the other frameworks? What do you think?
```suggestion
@Controller('/')
class BasicController {
  @Get()
  findAll() {
    return { error: null, framework: 'Nest fastify' };
  }
}

@Module({ imports: [], controllers: [BasicController], providers: [] })
```

Reply:
Good idea
To be honest, I did not know how to make a route.